### PR TITLE
#662(2) Change Request styles

### PR DESCRIPTION
**Pull request by CarlosNZ**
Updates the change request styles to match the latest layout and style changes. Should look pretty much the same as before (with the coloured borders around the elements).

Screen Shot 2021-05-09 at 10 34 31 PM

@nmadruga -- I have tried to preserve the conditional logic for the colour/style display in ApplicationViewWrapper using simple class switching rather than inline styles, but it'd be worth having a careful look at that bit to make sure.

- Also updated the "Make Changes" style in Applications List.

**NOTE:** There is still some work to do to make these elements on the Summary page consistent with on the Review page (re-using styles, and maybe even components)

### CODE REVIEW

**Review by nmadruga — CHANGES_REQUESTED** [review]
Very nice. It looks way tidier like that.

I have noticed one tiny issue: I think the colour for the icon here should be blue:
Screen Shot 2021-05-10 at 10 59 49 PM

Also there is something different on the `info` icon  regarding alignment, it looks different from the alert one for some reason:
Screen Shot 2021-05-10 at 11 01 07 PM

File: src/components/List/Cells/StatusCell.tsx [hunk 1]
```diff
@@ -11,27 +11,37 @@ enum ACTIONS {
...
     case ApplicationStatus.Draft:
       return (
         <Segment basic textAlign="center">
           <Progress size="tiny" />
-          <Link to={`/application/${serial}`}>{ACTIONS.EDIT_DRAFT}</Link>
-          <Icon name="trash alternate outline" style={{ marginLeft: 10 }} />
+          <Link to={`/application/${serial}`} className="user-action">
+            {ACTIONS.EDIT_DRAFT}
+          </Link>
+          <Icon name="trash alternate outline" />
```
[the remark this answers is not in the document]
Comment (nmadruga) [reply src/components/List/Cells/StatusCell.tsx:30]: Hmm actually something is still missing, this "edit draft" should be showing with similar alignment to "make changes":

Screen Shot 2021-05-10 at 11 04 19 PM
Screen Shot 2021-05-10 at 11 03 34 PM

**Comment by CarlosNZ** [comment]
>  have noticed one tiny issue: I think the colour for the icon here should be blue:
> Screen Shot 2021-05-10 at 10 59 49 PM

Yes, good catch. Thanks. Turns out the logic was correct (same as yours previously), but hadn't defined the "blue" class. 🙄

Have called the class "interactive-color", as I think "blue" will clash with existing classes. 9f13ff04

Quick question: Do you think the text should be blue as well, or just the icon? Here it is with both:
Screen Shot 2021-05-11 at 9 36 21 AM

File: semantic/src/site/globals/site.overrides [hunk 2]
```diff
@@ -532,6 +532,27 @@ a:hover {
...
+.alert-border {
+  border-color: @attention;
+}
+
+.blue-border {
+  border-color: @interactiveHigh;
+}
+
+.reviewer-comment {
+  font-size: 0.9em;
```
[the remark this answers is not in the document]
Comment (CarlosNZ) [reply semantic/src/site/globals/site.overrides:554]: Okay, I've made some additional relative size text size variables. We can probably consolidate these down to just one or two at some point, but will wait until we see the whole thing in context. 62f9ccbd

**Comment by CarlosNZ** [comment]
> Also there is something different on the `info` icon regarding alignment, it looks different from the alert one for some reason:

I'm not seeing it. 😕 What are you referring to exactly?

File: src/components/List/Cells/StatusCell.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (CarlosNZ) [reply src/components/List/Cells/StatusCell.tsx]: Thanks, done: e5f84f6c

I notice that the "Trash" icon doesn't actually do anything yet. Have added a "TO-DO" in the code, but do you think we should maybe hide it for now?

File: src/formElementPlugins/ApplicationViewWrapper.tsx [hunk 3]
```diff
@@ -230,26 +227,25 @@ const ChangesToResponseWarning: React.FC<ChangesToResponseWarningProps> = ({
...
+  const visibilityClass = isChangeRequest || (isChanged && isValid) ? '' : 'invisible'
+  const colourClass = isChangeRequest ? (!isChanged ? 'alert' : '') : 'blue'
+  const icon = (
+    <Icon
+      name={
+        isChangeRequest
+          ? isChanged
+            ? 'comment alternate outline'
+            : 'exclamation circle'
+          : 'info circle'
```
[the remark this answers is not in the document]
Comment (CarlosNZ) [reply src/formElementPlugins/ApplicationViewWrapper.tsx]: Good idea, it's much cleaner now: 733d4ecf

**Comment by nmadruga** [comment]
> Quick question: Do you think the text should be blue as well, or just the icon? Here it is with both:
> Screen Shot 2021-05-11 at 9 36 21 AM

Looks great to me

**Comment by nmadruga** [comment]
> > Also there is something different on the `info` icon regarding alignment, it looks different from the alert one for some reason:
> 
> I'm not seeing it. 😕 What are you referring to exactly?

It seemed not aligned to me, but now all in blue it seems good, so probably nothing.

**Pull request merged**